resources/certificate.rb

```diff
@@ -5,15 +5,16 @@
 
 property :certfile, String
 property :cert_password, String, sensitive: true
-property :keychain, String
+property :keychain, String, required: true
+property :kc_passwd, String, required: true, sensitive: true
```

Comment:
👍 
Awesome, was thinking about doing similar on fork, but wasn't sure of use-case for `node['macos']['admin_password']`